src/SFA.DAS.AANHub.Application/Employers/Commands/CreateEmployerMember/CreateEmployerMemberCommand.cs

```diff
@@ -10,6 +10,7 @@ public class CreateEmployerMemberCommand : CreateMemberCommandBase, IRequest<Val
 {
     public long AccountId { get; init; }
     public Guid UserRef { get; init; }
+    public string NetworkHubLink { get; set; } = null!;
```

Comment:
Dont think we need this any more, all the links will be pointing to the same endpoint